src/core/redux/selectors/selectAccessRequestData.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import { List } from 'immutable';
+
+import { ACCESS } from '../constants';
+
+export default function selectAccessRequestData() {
+  return (state :Map) :List => state.getIn([ACCESS, 'accessRequest'], List());
```

Comment:
I've been omitting the `notSetValue` param in favor of `|| List()` in selectors because that guarantees the selector will return a value.